src/resources/account/Preferences.js

```diff
@@ -0,0 +1,36 @@
+import Resource from '../Resource'
+
+export default class Preferences extends Resource {
+  constructor (api) {
+    super(api)
+    this.endpoint = 'user/preferences'
+  }
+
+  async list () {
+    return await this.request({
+      method: 'GET',
+      path: this.endpoint
+    })
+  }
+
+  async retrieve (key) {
```

Comment:
Is there a reason why you did something different then that was described in the task? My idea was to allow 'bulk' getting as well. So something like:

```js
// Setting
await deskbookers.preferences.set({
   'key1': 'val1',
   'key2': 'val2'
})

// Getting
const { key1, key2 } = await deskbookers.account.preferences.get([ 'key1', 'key2' ])
```

This is way faster than doing something like:

```js
await deskbookers.account.preferences.set({
   key: 'key1',
   val: 'val1'
})
await deskbookers.account.preferences.set({
   key: 'key2',
   val: 'val2'
})
const key1 = await deskbookers.account.preferences.get('key1')
const key2 = await deskbookers.account.preferences.get('key2')
```

Maybe smart to support both ways though, so both `get(String key)` and `get(String[] keys)`